src/index.js

```diff
@@ -40,6 +44,23 @@ export const buildTheme = (themeName, customColors={}) => {
   }
 }
 
+const removePixels = core => {
+  Object.keys(core).forEach(key => {
+    const category = core[key]
+    if (
+      key === 'BorderRadius' ||
+      key === 'FontSize' ||
+      key === 'LineHeight' ||
+      key === 'Spacing' ||
+      key === 'MediaQuery'
+    ) {
+      Object.keys(category).forEach(property => {
+        category[property] = parseInt(category[property], 10)
+      })
+    }
+  })
+}
+
```

Comment:
I don't like this approach and it doesn't solve the problem on all the attributes. 

```
parseInt('20px 10px', 10) === 20
```

I also asked Denna for her opinion on the `px` definition and she also was leaning towards not including it. So that's two environments that prefer to not have it.

Here is a simple util function that can be used to add the `px` like you're looking for. 

```
const addPx = (input) => {
  if (typeof input === 'number') {
    return `${input}px`
  }

  return input.split(' ').map(v => `${v}px`).join(' ')
}
```

Reply:
oh, right. I totally forgot about the tokens that have 2 values. Are you saying that you don't want to add this to the token repo? Or just I should be adding the `px` back in instead of removing them?

Reply:
I think the default should be no `px` with 2 of the 3 consumers preferring not to have it. You could use that function above to add it to both use cases. From my quick testing, the only singular value that seems to require the px for React is `lineHeight` btw.